code/modules/atmospherics/machinery/portable/portable_atmospherics.dm

```diff
@@ -93,16 +101,25 @@
 	. = ..()
 	if(!istype(user) || !user.canUseTopic(src, BE_CLOSE, NO_DEXTERITY, FALSE, !iscyborg(user)) || !can_interact(user))
 		return
-	if(holding)
-		to_chat(user, "<span class='notice'>You remove [holding] from [src].</span>")
-		replace_tank(user, TRUE)
+	if(!holding)
+		return
+	to_chat(user, "<span class='notice'>You remove [holding] from [src].</span>")
+	replace_tank(user, TRUE)
 
 /obj/machinery/portable_atmospherics/examine(mob/user)
 	. = ..()
-	if(holding)
-		. += "<span class='notice'>\The [src] contains [holding]. Alt-click [src] to remove it.</span>"+\
-			"<span class='notice'>Click [src] with another gas tank to hot swap [holding].</span>"
-
+	if(!holding)
+		return
+	. += "<span class='notice'>\The [src] contains [holding]. Alt-click [src] to remove it.</span>"+\
+		"<span class='notice'>Click [src] with another gas tank to hot swap [holding].</span>"
```

Comment:
Try to avoid changes like this one in the future. You're actually increasing the LoC used here.

Reply:
ah sure, if you want i can revert this